dspace-api/src/main/java/org/dspace/core/Email.java

```diff
@@ -396,11 +411,9 @@ void build()
         // Create message
         message = new MimeMessage(session);
 
-        // Set the recipients of the message
-        for (String recipient : recipients) {
-            message.addRecipient(Message.RecipientType.TO,
-                    new InternetAddress(recipient));
-        }
+        // Get the mail configuration properties for fixedRecipient
+        String[] fixedRecipients = getFixedRecipients(config);
```

Comment:
I would prefer to keep the existing code of doing a cycle for any configured recipients. Just adding a validation for `mail.catchAllAccount.enabled = true` to use the catchAll alternative instead.